imgparse/imgparse.py

```diff
@@ -670,6 +669,37 @@ def get_bandnames(image_path, exif_data=None, xmp_data=None):
     try:
         make, model = get_make_and_model(image_path, exif_data)
         xmp_tags = xmp.get_tags(make)
-        return parse_seq(xmp_data[xmp_tags.BANDNAME])
+        try:
+            return parse_seq(xmp_data[xmp_tags.BANDNAME])
+        except TypeError:
+            return [xmp_data[xmp_tags.BANDNAME]]
     except KeyError:
         raise ParsingError("Couldn't parse bandnames. Sensor might not be supported")
+
+
+@get_if_needed("exif_data", getter=get_exif_data, getter_args=["image_path"])
+def get_lens_model(image_path, exif_data=None):
```

Comment:
Add a test for this as well

Reply:
https://github.com/SenteraLLC/py-image-metadata-parser/pull/63/commits/128f96f0bb5a2d923c00bdf2fdef61f780c3febb Added tests for each Sentera camera in there, added a test for DJI to make sure it raises a parsing error https://github.com/SenteraLLC/py-image-metadata-parser/pull/63/commits/1c392612969ffd500be8efdeba7938459a4a1833